blog/2024-07-31-open-planning-r24.12.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Tractus-X Open Planning R24.12
+description: The Eclipse Tractus-X Open Planning Days - Empowering the Catena-X Data Space
+slug: open-planning-r24-12
+date: 2024-04-16T11:00
+hide_table_of_contents: false
+authors:
```

Comment:
I guess we could introduce the `authors.md` file for the news block

Reply:
Good idea, but I would like to move it to another PR. This here is just related to content. 

_Note: If I saw it correctly, the author file is right now persisted in the meeting-minutes folder. Wouldn't like to refer just this file. In my opinion, it should be stored on another level._

Reply:
Totally fine. The authors.yaml can be configured per blog instance. This is the [meeting minute one](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L102). 
I bet the same setting should be available for the "build in" blog plugin instance [here](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L50)